### PR TITLE
pythonPackages.sqlite-utils: init at 2.22

### PR DESCRIPTION
**Pull request by meatcar**
###### Motivation for this change

Adding a useful tool for tweaking sqlite dbs.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/sqlite-utils/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  checkInputs = [
+    pytest
+    pytestrunner
+    black
+  ];
+
+  checkPhase = ''
+    pytest
+  '';
```
Comment (jonringer) [inline pkgs/development/python-modules/sqlite-utils/default.nix:37]: personal preference
```suggestion
  checkInputs = [
    pytestCheckHook
    pytestrunner
    black
  ];
```

File: pkgs/development/python-modules/sqlite-utils/default.nix [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, click
+, click-default-group
+, tabulate
+, pytest
```
Comment (jonringer) [inline pkgs/development/python-modules/sqlite-utils/default.nix:8]:
```suggestion
, pytestCheckHook
```

File: pkgs/development/python-modules/sqlite-utils/default.nix (hunk 1 above)
Comment (meatcar) [reply pkgs/development/python-modules/sqlite-utils/default.nix]: Thanks, didn't know about `pytestCheckHook`, change applied.

**Comment by meatcar** [comment]
@jonringer thanks for the review, I implemented your suggestions. Also bumped up to v2.22 that was released since.

Is there anything else I should do to help this PR get merged?

**Review by Pamplemousse — APPROVED** [review]
LGTM :+1:
Built and tested on NixOS: application starts and displays help, library imported without error in Python.

**Pull request merged**